docs/docs/core/useAnimatedStyle.mdx

```diff
@@ -58,6 +60,12 @@ Only relevant when using Reanimated [without the Babel plugin on the Web](https:
 
 `useAnimatedStyle` returns an animated style object which has to be passed to the `style` property of an [Animated component](/docs/fundamentals/glossary#animated-component) that you want to animate.
 
+Types of `useAnimatedStyle` are made so they mimic behavior of `StyleSheet` styles as much as possible - `updater` callback return value should just look like a regular style object - with the exception of using shared values.
```

Comment:
```suggestion
`useAnimatedStyle` mimics the behavior of `StyleSheet` as much as possible. `updater` callback returns a value that looks like a regular style object in which you can also use shared values.
```
A bit simplified. You can rewrite this sentence in another way if you want but try to keep it simple 🙏